services/slam/fake/data_loader.go

```diff
@@ -34,9 +34,8 @@ type extra struct {
 }
 
 type position struct {
-	Pose               pose   `json:"pose"`
-	ComponentReference string `json:"component_reference"`
-	Extra              extra  `json:"extra"`
+	Pose  pose  `json:"pose"`
+	Extra extra `json:"extra"`
```

Comment:
do we still need the Extra here?

Reply:
yeah I wasn't sure about this but it's used in the `fakePosition` function. was hoping @JohnN193 has some insight on how necessary it is

Reply:
i don't remember if the positions in artifact use the extra field. I believe they do not but I would have to double check.

they should be in `slam/example_cartographer_outputs/viam-office-02-22-3/position ` or something like that. I'll be able to look myself later today

Reply:
I'm not sure where this file is

Reply:
![Screenshot 2024-05-22 at 11 21 18 AM](https://github.com/viamrobotics/rdk/assets/92045055/fd20c98d-d545-4a09-afc1-6b6070b617db)
no worries, basically we use artifact to store any files we need for testing & for fake components/services. More info can be found [here](https://viam.atlassian.net/wiki/spaces/ENG/pages/262371/Viam+Engineering+Guidelines#Testing-with-big-data)

it does look like the fake datasets still use the extras field, so we should not remove that from this pr. Might be worth uploading a new dataset someday but I do not believe thats necessary for this pr.

Reply:
okay sounds good I'll leave it